R/pagination.R

```diff
@@ -1082,8 +1148,16 @@ paginate_to_mpfs <- function(obj,
                              pg_size_spec = NULL,
                              page_num = default_page_number(),
                              rep_cols = NULL,
+                             # rep_cols = num_rep_cols(obj),
+                             # col_gap = 3, # this could be change in default - breaking change
                              col_gap = 2,
```

Comment:
What is the standard? Is it 3? We should align all defaults to 3 or 2

Reply:
Thats what I'm saying here, the defaults didn't match around the codebase. 2 seemed more common than 3, so I changed this 3 to 2.

I agree we should make sure the default is the same everywhere.

Reply:
This comment is coming from my merge, I think you changed this default to 3 in this PR. For me it is the same ;)